ledger-core/virtual/callsummary/callsummary.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+//go:generate sm-uml-gen -f $GOFILE
+
+package callsummary
+
+import (
+	"github.com/insolar/assured-ledger/ledger-core/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/pulse"
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/vanilla/injector"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/callregistry"
+)
+
+// key for publish synclink, need for await publish call results after migration for specific object
```

Comment:
godoc style is prefered 
// SummarySyncKey is used ...

Reply:
Fixed